core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/rds/responses/RdsTestResponse.java

```diff
@@ -1,22 +1,22 @@
-package com.sequenceiq.cloudbreak.api.model.rds;
+package com.sequenceiq.cloudbreak.api.endpoint.v4.rds.responses;
 
 import com.sequenceiq.cloudbreak.api.model.JsonEntity;
 import com.sequenceiq.cloudbreak.doc.ModelDescriptions;
 
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 
-@ApiModel("RdsTestResult")
-public class RdsTestResult implements JsonEntity {
+@ApiModel("RdsTestResponse")
```

Comment:
do we need RdsTestResponse here ?

Reply:
yes, it is needed for the testconnection endpoint